### DISCUSSION TITLE
Moved Immich to new host, now getting "Failed to get server config (500)" error - Help

### DISCUSSION THREAD

**Discussion by captainkev76** [immich-app/immich#9154] [Q&A]
Hi! My Docker host machine (Ubuntu 22.04) has had a bit of a meltdown so I've had to do a fresh build (on 24.04 now). Before the old box gave up the ghost I managed to backup the docker directory (/var/lib/docker). The media folders were already on a network drive, so I hoped it would be as simple as starting the docker container on the new host after copying the /var/lib/docker) files across. However, I now seem to get the following errors when I go to the homepage of my Immich instance.

```
Failed to get server config (500)
Stacktrace

he@http://192.168.1.94:2283/_app/immutable/chunks/fetch-client.DxqhYdA9.js:1:2911
de@http://192.168.1.94:2283/_app/immutable/chunks/fetch-client.DxqhYdA9.js:1:2835

```

Please can anyone advise what the issue may be?

**Comment by captainkev76** [comment]
Sorted it out! It was a permissions thing (isn't it ALWAYS on Ubuntu) - changed the permissions, restarted the stack and it's all good

**Reply by taltoris** [reply]
I'm getting this issue. Which folders did you change permissions on?

**Reply by captainkev76** [reply]
I can't actually remember now. It can only be one of two folders - either the media folder or the docker volume folder. Pretty sure it was the media folder

**Reply by bharathcs** [reply]
I had the same issue, and your comment helped me fix it -- just went:

```sh
docker compose down
chown -R root . # assuming all relevant paths are in CWD
docker compose up -d
```

For a while though, i kept getting errors on the immich frontend, but it cleared up after clearing cache (no idea why that should affect it).
